int64 integtime
int64 repeat
float64 synctime
float64 rugtime
string from_node
float64 timestamp
